test/test_jit.py

```diff
@@ -12703,28 +12720,6 @@ def fn(x, k):
         # the same group; they should each be a separate DiffGraph
         self.assertGraphContainsExactly(graph, 'prim::DifferentiableGraph', 2)
 
-    def test_mutation_subgraph_inlining(self):
```

Comment:
Why did this test disappear? :)

Reply:
See the accompanied change in create_autodiff_subgraphs.cpp.